packages/ui/src/memberships/components/MemberProfile/MemberAccounts.tsx

```diff
@@ -1,40 +1,68 @@
 import React from 'react'
 import styled from 'styled-components'
 
+import { useBalance } from '@/accounts/hooks/useBalance'
+import { TokenValue } from '@/common/components/typography'
+import { isDefined } from '@/common/utils'
+
 import { UnknownAccountInfo } from '../../../accounts/components/UnknownAccountInfo'
-import { AccountRow } from '../../../common/components/Modal'
 import { RowGapBlock } from '../../../common/components/page/PageContent'
 import { SidePaneLabel } from '../../../common/components/SidePane'
 import { Member } from '../../types'
 
-export const MemberAccounts = ({ member }: { member: Member }) => (
-  <AccountsDisplay gap={16}>
-    <SidePaneLabel text="Root account" />
-    {!!member.rootAccount && (
-      <AccountRow>
-        <UnknownAccountInfo address={member.rootAccount} placeholderName="Root Account" />
-      </AccountRow>
-    )}
-    <SidePaneLabel text="Controller account" />
-    {!!member.controllerAccount && (
-      <AccountRow>
-        <UnknownAccountInfo address={member.controllerAccount} placeholderName="Controller Account" />
-      </AccountRow>
-    )}
-
-    {!!member.boundAccounts.length && (
-      <>
-        <SidePaneLabel text="Bound accounts" />
-        {member.boundAccounts.map((account) => (
-          <AccountRow key={account}>
-            <UnknownAccountInfo address={account} placeholderName="Bound Account" />
-          </AccountRow>
-        ))}
-      </>
-    )}
-  </AccountsDisplay>
-)
+import { MemberBounds } from './MemberBounds'
+
+export const MemberAccounts = ({ member }: { member: Member }) => {
+  const rootBalance = useBalance(member.rootAccount)
+  const countrolBalance = useBalance(member.controllerAccount)
+
+  return (
+    <AccountsDisplay gap={16}>
+      <SidePaneLabel text="Root account" />
+      {!!member.rootAccount && (
+        <AccountMemberRow>
+          <UnknownAccountInfo address={member.rootAccount} placeholderName="Root Account" />
+          <TokenValue value={rootBalance?.total} isLoading={!isDefined(rootBalance?.total)} />
+        </AccountMemberRow>
+      )}
+      <SidePaneLabel text="Controller account" />
+      {!!member.controllerAccount && (
+        <AccountMemberRow>
+          <UnknownAccountInfo address={member.controllerAccount} placeholderName="Controller Account" />
+          <TokenValue value={countrolBalance?.total} isLoading={!isDefined(countrolBalance?.total)} />
+        </AccountMemberRow>
+      )}
+
+      {!!(member.boundAccounts.length !== 0) && (
+        <>
+          <SidePaneLabel text="Bound accounts" />
+          {member.boundAccounts.map((account) => {
+            return <MemberBounds account={account} />
+          })}
+        </>
+      )}
+    </AccountsDisplay>
+  )
+}
 
 const AccountsDisplay = styled(RowGapBlock)`
   padding: 24px;
 `
+
+export const AccountMemberRow = styled.div`
```

Comment:
Why do we need to duplicate this style? We can use `AccountRow` as was imported before, and overwrite necessary CSS with `styled(AccountRow)`